StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -207,19 +207,26 @@ extension STPAPIClient {
         var parameters: [String: Any] = [
             "credentials": ["consumer_session_client_secret": consumerSessionClientSecret],
             "request_surface": "ios_payment_element",
+            "expand": ["payment_method"],
```

Comment:
The ticket called for sending `expected_payment_method_type` but not sending it also works, which is good b/c IDK how we can tell what the value should be.